lld/ELF/OutputSections.h

```diff
@@ -137,6 +137,25 @@ struct OutputDesc final : SectionCommand {
   }
 };
 
+// A list of input sections that can be referenced in output descriptions.
```

Comment:
// This represents a `CLASS(class_name) { ... }` that can be referenced by output section descriptions. If referenced more than once, the sections can be spilled to the next reference like --enable-non-contiguous-regions.

Reply:
Done.